src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -296,7 +295,11 @@ export async function startTransformByQ() {
                 )
             }
             if (transformByQState.isSucceeded()) {
-                vscode.window.showInformationMessage(CodeWhispererConstants.transformByQCompleted)
+                vscode.window.showInformationMessage(CodeWhispererConstants.transformByQCompleted, { modal: true })
+            } else if (transformByQState.isPartiallySucceeded()) {
+                vscode.window.showInformationMessage(CodeWhispererConstants.transformByQPartiallyCompletedMessage, {
+                    modal: true,
```

Comment:
keeping this as a modal for consistency with the other information message above